src/test/java/com/jcabi/log/VerboseProcessTest.java

```diff
@@ -236,14 +244,163 @@ public void logsErrorWhenUnderlyingStreamIsClosed() throws Exception {
+                    return null;
+                }
+            }
+        ).when(prc).destroy();
+        final VerboseProcess verboseProcess = new VerboseProcess(
+            prc,
+            Level.FINEST,
+            Level.FINEST
+        );
+        final StringWriter writer = new StringWriter();
+        final WriterAppender appender = new WriterAppender(
+            new SimpleLayout(),
+            writer
+        );
+        appender.addFilter(new VrbPrcMonitorFilter(verboseProcess));
+        org.apache.log4j.Logger.getLogger(
+            VerboseProcess.class
+        ).addAppender(appender);
+        final int readInterval = 50;
+        new Timer(true).schedule(
+            new TimerTask() {
+                @Override
+                public void run() {
+                    verboseProcess.close();
+                }
+            },
+            readInterval
+        );
+        verboseProcess.stdoutQuietly();
+        final int monitorsInterval = 500;
+        TimeUnit.MILLISECONDS.sleep(monitorsInterval);
+        Mockito.verify(
+            prc,
+            Mockito.atLeastOnce()
+        ).destroy();
+        MatcherAssert.assertThat(
+            writer.toString(),
+            Matchers.not(Matchers
+                    .containsString("Error reading from process stream")
+            )
+        );
+    }
+
+    /**
+     * {@link InputStream} returning endless flow of characters.
+     */
+    private final class InfiniteInputStream extends InputStream {
+        /**
+         * End of line.
+         */
+        private static final int LINE_FEED = 0xA;
+
+        /**
+         * Character, endlessly repeated in the stream.
+         */
+        private final transient char chr;
+        /**
+         * Whether the next char in the stream should be EOL.
+         */
+        private transient boolean feed;
+        /**
+         * Whether this stream is closed.
+         */
+        private transient boolean closed;
+
+        /**
+         * Construct an InputStream returning endless combination of this
+         * character and end of line.
+         * @param character Character to return in the stream
+         */
+        public InfiniteInputStream(final char character) {
+            super();
+            this.chr = character;
+        }
+
+        @Override
+        public int read() throws IOException {
+            if (this.closed) {
+                throw new IOException("Stream closed");
+            }
+            final int next;
+            if (this.feed) {
+                this.feed = false;
+                next = LINE_FEED;
+            } else {
+                this.feed = true;
+                next = this.chr;
+            }
+            return next;
+        }
+
+        @Override
+        public void close() throws IOException {
+            this.closed = true;
+        }
+    }
+
+    /**
+     * Filter of log messages of {@link VerboseProcess}'s monitor threads.<br/>
```

Comment:
sorry, haven't got. What to add? If just line break then we will not see it in the javadoc.
Afaik, we can leave <br/> or use <p> on the next line to have the break in the javadoc.